### ISSUE TITLE
 What method is used in version 1.2.0 to re-construct TCP flow and extract data from TCP packets in the right order

### ISSUE THREAD

**Issue by Stool233** [buger/goreplay#857]
What method is used in version 1.2.0 to re-construct TCP flow and extract data from TCP packets in the right order？
I didn't find the relevant code. 
Please tell me what method is used in version 1.2.0 or where the relevant code is？

**Comment by urbanishimwe** [comment]
https://github.com/buger/goreplay/blob/master/tcp/tcp_message.go

**Comment by Stool233** [comment]
> https://github.com/buger/goreplay/blob/master/tcp/tcp_message.go

Is this method？
![image](https://user-images.githubusercontent.com/21293711/100197223-ff1fa180-2f34-11eb-9410-cac7a75c6f45.png)

But I didn't find other code that called this method

**Comment by urbanishimwe** [comment]
yes, it is never used. Ideally, packets of TCP should be ordered. It's just there for external consumers.

**Comment by buger** [comment]
Do you have any specific use-case/idea in mind?
